src/screens/DataApi/DataCoverageTable.tsx

```diff
@@ -0,0 +1,172 @@
+import { InfoTooltip } from 'components/InfoTooltip';
+import { pluralize } from 'components/Explore/utils';
+import { RegionType } from 'common/regions';
+
+export const COVERAGE_SUMMARY = coverageSummary as MetricCoverage[];
+
+interface MetricField {
+  displayName: string;
+  field: string;
+}
+
+export const FIELDS: MetricField[] = [
+  { displayName: 'Overall risk level', field: 'riskLevels.overall' },
+  { displayName: 'Daily new cases', field: 'metrics.caseDensity' },
+  { displayName: 'Infection rate', field: 'metrics.infectionRate' },
+  { displayName: 'Positive test rate', field: 'metrics.testPositivityRatio' },
+  { displayName: '% Vaccinated', field: 'metrics.vaccinationsInitiatedRatio' },
+  { displayName: 'ICU capacity used', field: 'metrics.icuCapacityRatio' },
+  { displayName: 'Hospitalization usage', field: 'metrics.icuCapacityRatio' },
+  { displayName: 'Deaths', field: 'actuals.deaths' },
+  { displayName: 'Total testing volume', field: 'actuals.positiveTests' },
+];
+
+export const FIELDS_PRETTY_NAMES = FIELDS.map(field => {
+  const record = COVERAGE_SUMMARY.find(record => field.field === record.name);
+  if (record) {
+    return {
+      ...record,
+      ...{ name: field.displayName },
+    };
+  }
+
+  return null;
+}).filter((record): record is MetricCoverage => record !== null);
+
+export interface CoverageDetails {
+  totalRegions: number;
+  regionsAvailable: number;
+  totalStates: number;
+  statesAvailable: number;
+}
+
+export interface MetricCoverage {
+  name: string;
+  state: CoverageDetails;
+  county: CoverageDetails;
+  metro: CoverageDetails;
+}
+
+const TextWithTooltip: React.FC<{ text: string; tooltipContent: string }> = ({
+  text,
+  tooltipContent,
+}) => {
+  return (
+    <TextWithTooltipContainer>
+      <UnderlinedText align="center">
+        {text}
+        <InfoTooltip trackOpenTooltip={() => {}} title={tooltipContent} />
+      </UnderlinedText>
+    </TextWithTooltipContainer>
+  );
+};
+
+const tooltipText = (coverage: CoverageDetails, regionType: RegionType) => {
+  switch (regionType) {
+    case RegionType.COUNTY:
+      return (
+        `Supported in ${coverage.regionsAvailable} counties (of ${coverage.totalRegions}), ` +
+        `or ${coverage.statesAvailable} states and territories (of ${coverage.totalStates}).`
+      );
+    case RegionType.MSA:
+      return `Supported in ${coverage.regionsAvailable} metros (of ${coverage.totalRegions}).`;
+    case RegionType.STATE:
+      return `Supported in ${coverage.regionsAvailable} states (of ${coverage.totalRegions}).`;
+  }
+};
+
+const Status: React.FC<{ status: CoverageDetails; level: RegionType }> = ({
+  status,
+  level,
+}) => {
+  if (status.regionsAvailable === 0) {
+    return (
+      <TextWithTooltip text="No data" tooltipContent="No data available." />
+    );
+  }
+
+  const availableRatio = status.regionsAvailable / status.totalRegions;
+
+  if (availableRatio >= 0.95) {
+    return <CheckIcon color="secondary" />;
```

Comment:
Consider adding the `aria-label` attribute to the icons to ensure that the information is available for users that need assistive technologies